### PR TITLE
[MASFoundation] ChangeLog update for 1.7.10

### PR DESCRIPTION
**Pull request by jhoongo**

### CODE REVIEW

**Comment by ghost** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qa-mas-mac-mini-1.ca.com/21fa19359da47bff9d6f4bb33437c0ff/21fa19359da47bff9d6f4bb334387394)
---
*Duration*: 3 minutes and 55 seconds
*Result*: All 0 tests passed, but please **fix 35 warnings** and **20 analyzer warnings**.

**Review by BobbyWeber — APPROVED** [review]
Approved

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-# Version 1.8.00
+# Version 1.7.10
 
 ### Bug fixes
 - MASFoundation's OTP channel selection was returning an internal error to the original request. The Mobile SDK now sends only cancellation of OTP process, and/or results of the original request. [DE366491]
+- The Mobile SDK was causing an error during device registration, "The device has already been registered." which prevents users to re-register the device using different credentials when share keychain was incorrectly configured. The Mobile SDK improves the device registration process to minimize this type of error. [DE369778]
```
Comment (bmckercher) [inline CHANGELOG.md:5]: During device registration, the  Mobile SDK was causing an error: "The device has already been registered".  This happened when the share keychain was incorrectly configured, preventing users from re-registering the device using different credentials. The Mobile SDK improved the device registration process to minimize this kind of error.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+- The Mobile SDK reserved "Content-Type" and "Accept" header for requests and did not provide an option for developers to override these values. The Mobile SDK now allows developers to override the values as needed. [DE369138]
```
Comment (bmckercher) [inline CHANGELOG.md:6]: The Mobile SDK reserved "Content-Type" and "Accept" header for request**s, but** did not provide an option for developers to override these values. The Mobile SDK now allows developers to override the values as needed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+- The Mobile SDK caused an incorrect error for session lock using multiple applications using SSO. The Mobile SDK now handles session lock's status properly, and no longer causes the issue. [DE374443]
```
Comment (bmckercher) [inline CHANGELOG.md:7]: The Mobile SDK displayed an incorrect error for session lock when multiple applications are using SSO. The Mobile SDK now provides accurate session lock status.

**Review by bmckercher — COMMENTED** [review]
Edits for clarity.

**Pull request merged**